TombEngine/Game/people.cpp

```diff
@@ -81,7 +81,7 @@ short GunMiss(int x, int y, int z, short velocity, short yRot, short roomNumber)
 {
 	auto pos = GameVector(
 		LaraItem->Pose.Position.x + ((GetRandomControl() - 0x4000) << 9) / 0x7FFF,
-		LaraItem->Floor,
+		LaraItem->Floor - 128,
```

Comment:
Why -128 is introduced here?

Reply:
the bullet which miss the target was inside the floor not on it so i added that.